tests/unit/core/test_store.js

```diff
@@ -56,34 +57,33 @@ describe('core store middleware', () => {
     expect(_createLogger.called).toEqual(false);
   });
 
-  it('uses a placeholder store enhancer when devtools is not installed', () => {
-    const _window = {}; // devToolsExtension() is undefined
-    const enhancer = middleware({
-      _config: configForDev(true),
-      _window,
-    });
+  it('uses a placeholder store enhancer when devtools is not available', () => {
+    const _window = {}; // __REDUX_DEVTOOLS_EXTENSION__() is undefined
+    const enhancer = middleware({ _config: configForDev(true), _window });
+
     expect(typeof enhancer).toBe('function');
+
     const createStore = () => {};
     expect(typeof enhancer(createStore)).toBe('function');
   });
 
-  it('adds the devtools store enhancer in development', () => {
+  it('adds the devtools store enhancer when config enables it', () => {
     const _window = {
-      devToolsExtension: sinon.spy((createStore) => createStore),
+      __REDUX_DEVTOOLS_EXTENSION__: sinon.spy(),
     };
-    expect(typeof middleware({ _config: configForDev(true), _window })).toBe(
-      'function',
-    );
-    expect(_window.devToolsExtension.called).toEqual(true);
+    const _config = configForDev(true, { enableDevTools: true });
```

Comment:
since this is no longer dependent on `isDevelopment`, the test doesn't need to set that in the config.